doorway-ui-components/src/global/app-css.scss

```diff
@@ -25,6 +25,7 @@ main {
 .site-wrapper {
   display: flex;
   min-height: 100vh;
+  min-height: 100dvh;
```

Comment:
Prevents browser navigation from overlaying on top of UI

Reply:
Interesting. I've never seen the `dvh` before. Cool!